README.md

```diff
@@ -129,11 +129,70 @@ kubetest --test=true \
   --test_args=--node-os-distro=windows --ginkgo.focus=\\[Conformance\\]|\\[NodeConformance\\]|\\[sig-windows\\]|\\[sig-apps\\].CronJob --ginkgo.skip=\\[LinuxOnly\\]|\\[k8s.io\\].Pods.*should.cap.back-off.at.MaxContainerBackOff.\\[Slow\\]\\[NodeConformance\\]|\\[k8s.io\\].Pods.*should.have.their.auto-restart.back-off.timer.reset.on.image.update.\\[Slow\\]\\[NodeConformance\\]
 ```
 
+## Running unit test
+
+Note: This assumes the Windows node is running on GCE, but should be applicable
+to other platforms with slight changes.
```

Comment:
I don't see anything GCP-specific below (maybe it was here previously but was then removed). Just remove this note?

Reply:
Yep, removed.